sandbox/extension/resource_relationship_2021-08_16.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<?xml-stylesheet type='text/xsl' href='/style/human.xsl'?>
+<extension xmlns='http://rs.gbif.org/extension/'
+    xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
+    xmlns:dc='http://purl.org/dc/terms/'
+    xsi:schemaLocation='http://rs.gbif.org/extension/ http://rs.gbif.org/schema/extension.xsd'
+    dc:title="Darwin Core Resource Relationship" 
+    name="ResourceRelationship" namespace="http://rs.tdwg.org/dwc/terms/" 
+    rowType="http://rs.tdwg.org/dwc/terms/ResourceRelationship"
```

Comment:
See https://github.com/tdwg/dwc/commit/1738b308f6c5584278b6670fab0b6875f48ee1d9 which changes the templates to use `'` throughout, just to make diffing easier.

Reply:
Amended throughout.